seam/seam.py

```diff
@@ -24,6 +25,7 @@ def __init__(
         workspace_id: Optional[str] = None,
         endpoint: Optional[str] = None,
         wait_for_action_attempt: Optional[Union[bool, Dict[str, float]]] = True,
+        retry_config: Optional[Retry] = None,
```

Comment:
Maybe just this, since it seems like `retries` is the Python standard name for passing this option around.
```suggestion
        retries: Optional[Retry] = None,
```

Reply:
8ed2e7058ee1916445856016db9a16780d97b5ec